### DISCUSSION TITLE
Welcome to The Web3 Index Discussions!

### DISCUSSION THREAD

**Discussion by adamsoffer** [web3index/web3index-org#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by johncodesit** [comment]
Hi @adamsoffer @baldator @Redm4x  @shreyaspapi !
Is there a timeline of proposed features/ideas for the web index? 
I want to contribute somehow with my web skills, sounds like a great usefull project :)

**Comment by adamsoffer** [comment]
Welcome @johncodesit! We'll be sharing a roadmap/vision soon. In the mean time, are there any features or ideas you'd like to see implemented in The Web3 Index? We're always seeking new ideas and contributions.

**Comment by Siegfried-B** [comment]
Is anyone still working on the Web3index?
At least a few data feeds (Helium and Arweave) haven't been updating on the website for several days.
Questions regarding this problem on the project's Discord also haven't been answered yet.
